Reading List/Reading List/Views/BookTableViewCell.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  BookTableViewCell.swift
+//  Reading List
+//
+//  Created by Nathanael Youngren on 1/17/19.
+//  Copyright © 2019 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class BookTableViewCell: UITableViewCell {
+    
+    func updateViews() {
+        if let book = book {
+            bookLabel.text = book.title
+            let read = book.hasBeenRead
+            
+            if read {
+                let checked = UIImage(named: "checked")
+                bookButton.setImage(checked, for: .selected)
```

Comment:
And this should be `.normal` state
```suggestion
                bookButton.setImage(checked, for: .normal)
```